github/serverless/deploy.yml

```diff
@@ -14,67 +16,100 @@ env:
   DAGSTER_CLOUD_API_TOKEN: ${{ secrets.DAGSTER_CLOUD_API_TOKEN }}
   ENABLE_FAST_DEPLOYS: 'true'
   PYTHON_VERSION: '3.8'
+  DAGSTER_PROJECT_DIR: '.'
   DAGSTER_CLOUD_FILE: 'dagster_cloud.yaml'
+  DAGSTER_CLOUD_ORGANIZATION: ${{ secrets.DAGSTER_CLOUD_ORGANIZATION }}
```

Comment:
This will break functionality on non-prod tiers, right? (e.g. dogfooding, canary). I'm hesitant to remove for this reason - we don't have a hybrid quickstart UI flow so less pressing